### ISSUE TITLE
Error while analyzing build.

### ISSUE THREAD

**Issue by ghuntley** [KirillOsenkov/MSBuildStructuredLog#262]
:wave: Here's the binlog - https://github.com/unoplatform/uno/files/3702207/msbuild.zip

```
Error while analyzing build. Sorry about that. Please Ctrl+C to copy this text and file an issue on https://github.com/KirillOsenkov/MSBuildStructuredLog/issues/new 

System.NullReferenceException: Referencia a objeto no establecida como instancia de un objeto.

   en Microsoft.Build.Logging.StructuredLogger.CompilationWrites.TryParse(Task task)

   en Microsoft.Build.Logging.StructuredLogger.ManagedCompilerTask.get_CompilationWrites()

   en Microsoft.Build.Logging.StructuredLogger.DoubleWritesAnalyzer.AnalyzeTask(Task task)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeTask(Task task)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Visit(TreeNode node)

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.Analyze()

   en Microsoft.Build.Logging.StructuredLogger.BuildAnalyzer.AnalyzeBuild(Build build)
---------------------------
Aceptar   
---------------------------
```

**Comment by KirillOsenkov** [comment]
Hey @ghuntley what's the viewer version? The log opens fine in 2.0.112. Try starting the viewer from the Start Menu, wait for it to auto-update, then restart again.

I remember fixing this, but it could be a new issue that I just can't repro.

**Comment by ghuntley** [comment]
I pulled msbuild viewer off chocolatey last night on a fresh computer.
@superjmn will have specifics as to version.

**Comment by KirillOsenkov** [comment]
oh, I haven't updated the choco one in a while... I should do that

**Comment by KirillOsenkov** [comment]
I'm almost positive that it's just a very old version

**Comment by KirillOsenkov** [comment]
I've published 2.0.113 to Chocolatey:
https://chocolatey.org/packages/msbuild-structured-log-viewer/2.0.113

it'll probably take a while for it to be approved. 
Use http://msbuildlog.com in the meanwhile to install it.

**Comment by KirillOsenkov** [comment]
I'll close this but feel free to reopen or file a new issue if it still reproduces with the latest viewer.

Also note that only English MSBuild is supported for now, as the viewer relies heavily on parsing English text to recover the structure from the log.

Long term I'd love to make the viewer work with logs in any language, but it's a lot of work.
